_docs/2.0.0/usage/menas-conformance-rules.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: docs
+title: Usage - Menas Conformance Rules
+version: '2.0.0'
+categories:
+    - '2.0.0'
+    - usage
+---
+
+## Table Of Contents
+<!-- toc -->
+- [Table Of Contents](#table-of-contents)
+- [Intro](#intro)
+- [Casting Conformance Rule](#casting-conformance-rule)
+- [Concatenation Conformance Rule](#concatenation-conformance-rule)
+- [Drop Conformance Rule](#drop-conformance-rule)
+- [Literal Conformance Rule](#literal-conformance-rule)
+- [Mapping Conformance Rule](#mapping-conformance-rule)
+- [Negation Conformance Rule](#negation-conformance-rule)
+- [SingleColumn Conformance Rule](#singlecolumn-conformance-rule)
+- [SparkSessionConf Conformance Rule](#sparksessionconf-conformance-rule)
+- [Uppercase Conformance Rule](#uppercase-conformance-rule)
+- [FillNulls Conformance Rule](#fillnulls-conformance-rule)
+<!-- tocstop -->
+
+## Intro
+
+Conformance Rules are a way for the user to enhance the data. Every conformance rule has an output column and an option for running a control measure. Output column defines the name of the output column into which the rule will output. Control measure check will run an Atum control measure check as in previouse stages as defined in controls of `_INFO` file.
+
+Every column also has one or more input columns.
+
+**Important** - We never override a column. Each rule produces a new column.
+
+## Casting Conformance Rule
+
+Allows users to cast a specific column to another type. This conformance rules provides a selection of other types.
+
```

Comment:
I think it would be nice to add some specifications. 
Like allowed "conversions"
- any type to string is ok 
- boolean to number is ok (false -> 0; true -> 1)
- number to boolean is ok  (0 -> false; any -> true)
- to binary type is only from string type